src/internal/git/fetch.go

```diff
@@ -48,3 +48,42 @@ func fetchTag(gitDirectory string, tag string) {
 		message.Fatal(err, "Not a valid tag or unable to fetch")
 	}
 }
+
+// FetchHash performs a `git fetch` of _only_ the provided commit hash.
+func FetchHash(gitDirectory string, hash string) {
```

Comment:
This function was made public but nothing outside of this package is calling it?